From a9e755b98968fa67d0b42d328ec4801c4b764077 Mon Sep 17 00:00:00 2001
From: Matthias Schoepfer <m.schoepfer@rethinkrobotics.com>
Date: Thu, 21 Sep 2023 13:05:27 +0200
Subject: [PATCH] Serialization.cpp: added missing include

For std::memcpy, the include of <cstring> is needed. If you have a
recent / picky compiler, it will complain.

This commit fixes it.

Signed-off-by: Matthias Schoepfer <m.schoepfer@rethinkrobotics.com>
---
 src/Serialization.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Serialization.cpp b/src/Serialization.cpp
index 4d1a015..15d3891 100644
--- a/src/Serialization.cpp
+++ b/src/Serialization.cpp
@@ -21,6 +21,7 @@
 #include "Serialization.hpp"
 
 #include <array>
+#include <cstring>
 #include <limits>
 #include <memory>
 #include <unordered_map>
-- 
2.41.0

